packages/vaadin-login/src/vaadin-login-mixin.js

```diff
@@ -70,6 +70,16 @@ export const LoginMixin = (superClass) =>
           notify: true
         },
 
+        /**
+         * If set, the user name field automatically receives focus when the component is attached to the document.
+         * @type {boolean}
+         */
+        autoFocusUsername: {
+          type: Boolean,
+          value: false,
+          notify: true
```

Comment:
`notify: true` is not needed here I think. According to docs it's purpose is to support two-way binding, which doesn't make sense here.

Reply:
Removed